org.envirocar.app/src/org/envirocar/app/views/preferences/BluetoothPairingPreference.java

```diff
@@ -257,11 +255,11 @@ private void updatePairedDevicesList() {
      * Initiates the discovery of other Bluetooth devices.
      */
     private void startBluetoothDiscovery() {
-        // If bluetooth is not enabled, skip the discovery and show a toast.
+        // If bluetooth is not enabled, skip the discovery and show a snackbar.
         if (!mBluetoothHandler.isBluetoothEnabled()) {
             LOGGER.debug("startBluetoothDiscovery(): Bluetooth is disabled!");
-            Toast.makeText(getContext(), "Bluetooth is disabled. Please enable Bluetooth before " +
-                    "discovering for other devices.", Toast.LENGTH_LONG).show();
+            showSnackbar("Bluetooth is disabled. Please enable Bluetooth before " +
+                    "discovering for other devices.");
```

Comment:
Maybe, you could use String resources for all these messages, too?

Reply:
sure will do it :)